docs/app/Examples/modules/Popup/Types/PopupHtmlExample.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Popup, Card, Rating, Image } from 'stardust'
+
+const users = [
+  {
+    name: 'Elliot Fu',
+    bio: 'Elliot has been a member since July 2012',
+    avatar: 'http://semantic-ui.com/images/avatar/small/elliot.jpg',
+  },
+  {
+    name: 'Stevie Feliciano',
+    bio: 'Stevie has been a member since August 2013',
+    avatar: 'http://semantic-ui.com/images/avatar/small/stevie.jpg',
+  },
+  {
+    name: 'Matt',
+    bio: 'Matt has been a member since July 2014',
+    avatar: 'http://semantic-ui.com/images/avatar/small/matt.jpg',
+  },
+]
+
+export default function () {
+  const IndividualCard = (
+    <Card>
+      <Image src='http://semantic-ui.com/images/movies/totoro-horizontal.jpg' />
+      <Card.Content>
+        <Card.Header>
+          My Neighbor Totoro
+        </Card.Header>
+        <Card.Description>
+          Two sisters move to the country with their father in order to be
+          closer to their hospitalized mother, and discover the surrounding
+          trees are inhabited by magical spirits.
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  )
+
+  const content = <div>
+    <div className='header'>User Rating</div>
+    <div className='content'>
+      <Rating icon='star' defaultRating={3} maxRating={4} />
+    </div>
+  </div>
+
+  return <Popup
+    trigger={IndividualCard}
+    content={content}
+    hover
+  />
```

Comment:
sounds good, I was also confused about title and header and was about to ask for confirmation :+1: